manager/integration/tests/test_stress.py

```diff
@@ -313,6 +315,24 @@ def restore_and_check_random_backup(client, core_api, volume_name, pod_name, sna
     assert bkp_checksum_ok
 
 
+def delete_and_wait_replica_rebuild(client, volume_name):
+    volume = client.by_id_volume(volume_name)
+
+    replica_count = len(volume.replicas)
+
+    replica_id = randrange(0, replica_count)
+
+    replica_name = volume["replicas"][replica_id]["name"]
+
+    volume.replicaRemove(name=replica_name)
+
+    wait_for_volume_replica_count(client, volume_name, replica_count)
+
+    replica_names = map(lambda replica: replica.name, volume["replicas"])
+
+    wait_new_replica_ready(client, volume_name, replica_names)
```

Comment:
- randomly decide if we want to wait to wait for replica to be rebuild.
- add a parameter to configure the chance to wait for an action to be completed.